libraries/vircadia-client/src/version.h

```diff
@@ -0,0 +1,65 @@
+//
+//  version.h
+//  libraries/client/src
+//
+//  Created by Nshan G. on 18 Feb 2022.
+//  Copyright 2022 Vircadia contributors.
+//
+
+/** \file */
+
+#ifndef VIRCADIA_LIBRARIES_CLIENT_SRC_VERSION_H
+#define VIRCADIA_LIBRARIES_CLIENT_SRC_VERSION_H
+
+#include "common.h"
+
+/**
+ * \brief Defines the layout of version information returned by vircadia_client_version().
+ */
+struct vircadia_client_version_data {
+
+    /** \brief The major version number.
+     *
+     * Increments on compatibility breaking changes.
+     */
```

Comment:
Is the client version of the form "2022.1.0"? If so, the doxygen needs updating: major = year; minor = breaking change; ...

Reply:
Well, this is not set in stone yet, just example of what it might look like. I'm not sure I understand that specific versioning scheme enough to document and use it properly.

Reply:
The versioning scheme used by Vircadia, the Web SDK, and the Web app is yyyy,m,n:
- yyyy = The year of the release.
- m = Major release. An increment in this indicates a breaking change (e.g., Vircadia protocol change).
- n = Minor release. Normal, non-breaking changes.

Reply:
Does the year change with either a major or a minor release, or do we wait for a major release to update it?

Reply:
The year changes whenever the first release of the new year occurs.

Reply:
Updated the code and the docs to reflect this, both here and in the unity sdk repo.